README.md

```diff
@@ -611,8 +662,59 @@ cc_cold_xsk1fp4megtpn4vu4cug2lmsyhg4xvnnar55q6k8wp5e6f2h8jz849ph8v8jhm0qffw8v6ut
 $ cardano-address key public --with-chain-code < cold.xsk > cold.xvk
 cc_cold_xvk1dg8d5du0v4ukqkfgset50xncudhwlfzz2p6epv096x0ndl8jsgzzdqzpe9yw37u7mu7xaenhv7242990ps3sn8jcg52yx7n3fuke9kst5t2py
 
-$ cardano-address key hash < cold.xvk > cold
+$ cardano-address key public --without-chain-code < cold.xsk > cold.vk
+cc_cold_vk1dg8d5du0v4ukqkfgset50xncudhwlfzz2p6epv096x0ndl8jsgzqmwj2x5
```

Comment:
I notice that we have `--with-chain-code` and `--without-chain-code` flags here, what is the default behavior if no chain code is specified?

Reply:
there is error if one does not specify this. One needs to explicitly state either `--with-chain` or `-without-chain`
https://github.com/IntersectMBO/cardano-addresses/blob/master/command-line/lib/Options/Applicative/Public.hs#L31